plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -8,7 +8,7 @@
 
 <div class="row">
     <div class="col s12">
-        <ul class="tabs" style="overflow: auto !important;height: auto !important;">
+        <ul class="tabs no-data-screen-ul-tabs" style="overflow: auto !important;height: auto !important;">
```

Comment:
would you mind moving the css in the style property to the less file as well?

Reply:
updated